.github/workflows/windows-build-and-test.yaml

```diff
@@ -56,6 +56,15 @@ jobs:
         tsl_ignores: ["compression_algos"]
         tsl_skips: ["bgw_db_scheduler bgw_db_scheduler_fixed"]
         pg_config: ["-cfsync=off -cstatement_timeout=60s"]
+        include:
+          - pg: 14
+            pg_version: 14.13
```

Comment:
Do we need both? Seems they can easily get out of sync leading to confusion. Can we parse out major version from the full version?

Reply:
The code for computing various things for builds is already very complicated, so this runs the risk of the "parsing" leading to more bugs and more difficult maintenance.

This approach is very clear and easy to set to whatever we consider as the latest version that we support, for anybody, not only developers. Explicitly updating this will also be seen in the commit history, which aids in debugging.